packages/apps-config/src/endpoints/production.ts

```diff
@@ -507,6 +507,17 @@ export const prodChains: Omit<EndpointOption, 'teleport'>[] = [
       logo: nodesPolymeshSVG
     }
   },
+  {
+    info: 'quantum-fusion',
```

Comment:
We don't use `quantum fusion` name anymore, maybe `qfnetwork`? Should we replace this in other places in the same patch?